content/guides/models/app/settings-page/billing-settings.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Manage your organization's billing settings
+menu:
+  default:
+    parent: settings
+title: Manage billing settings
+weight: 20
+---
+
+Navigate to your user profile page and select your user icon on the top right corner. From the dropdown, choose **Settings**, then click **Billing**.
```

Comment:
There is also an a direct **Billing** option that appears from the dropdown. Can save a step by omitting clicking on **Settings**.